server/platform/controllers/event_log.go

```diff
@@ -45,7 +45,11 @@ func (ec EventLogController) List(c *gin.Context) {
 		if err == nil {
 			identityIDs = append(identityIDs, identity.ID)
 		} else {
-			c.AbortWithStatus(http.StatusNoContent)
+			res := responses.EventLogList{
+				Events: []responses.EventLog{},
+				Count:  0,
+			}
+			c.JSON(http.StatusOK, &res)
```

Comment:
Why not return an empty `responses.EventLogList{}` as the `Count` field will always be 0.